Flaky build-pt

https://github.com/trinodb/trino/actions/runs/3519082265/jobs/5898653499

```
RuntimeError: ptl suite describe failed: CompletedProcess(args=['testing/bin/ptl', 'suite', 'describe', '--suite', 'suite-1,suite-2,suite-3,suite-5,suite-delta-lake-databricks73,suite-delta-lake-databricks91,suite-delta-lake-databricks104,suite-delta-lake-databricks113,suite-clients,suite-functions,suite-tpch,suite-storage-formats-detailed,suite-6-non-generic,suite-7-non-generic,suite-8-non-generic,suite-tpcds,suite-parquet,suite-oauth2,suite-ldap,suite-compatibility,suite-all,suite-delta-lake-oss,suite-kafka,suite-cassandra,suite-clickhouse,suite-mysql,suite-iceberg,suite-hudi', '--config', 'config-default', '--format', 'JSON'], returncode=1, stdout='', stderr="2022-11-22T01:34:51.114Z\tINFO\tmain\tio.airlift.log.Logging\tLogging to stderr\n2022-11-22T01:34:51.116Z\tINFO\tmain\tBootstrap\tLoading configuration\n2022-11-22T01:34:51.435Z\tINFO\tmain\tBootstrap\tInitializing logging\n2022-11-22T01:34:51.929Z\tINFO\tmain\tBootstrap\tPROPERTY  DEFAULT  RUNTIME  DESCRIPTION\n2022-11-22T01:34:52.458Z\tINFO\tmain\tio.airlift.bootstrap.LifeCycleManager\tLife cycle starting...\n2022-11-22T01:34:52.460Z\tINFO\tmain\tio.airlift.bootstrap.LifeCycleManager\tLife cycle started\n2022-11-22T01:34:52.629Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode with extender: Standard; options: {}\n2022-11-22T01:34:52.661Z\tWARN\tmain\torg.testcontainers.utility.TestcontainersConfiguration\tAttempted to read Testcontainers configuration file at file:/home/runner/.testcontainers.properties but the file was not found. Exception message: FileNotFoundException: /home/runner/.testcontainers.properties (No such file or directory)\n2022-11-22T01:34:52.665Z\tINFO\tmain\torg.testcontainers.utility.ImageNameSubstitutor\tImage name substitution will be performed by: DefaultImageNameSubstitutor (composite of 'ConfigurationFileImageNameSubstitutor' and 'PrefixingImageNameSubstitutor')\n2022-11-22T01:34:52.676Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode with extender: StandardMultinode; options: {}\n2022-11-22T01:34:52.677Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode with extender: Hadoop; options: {}\n2022-11-22T01:34:52.679Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode with extender: EnvMultinode; options: {}\n2022-11-22T01:34:52.679Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode with extender: ConfigDefault; options: {}\n2022-11-22T01:34:52.710Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode with extender: Standard; options: {}\n2022-11-22T01:34:52.711Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode with extender: Hadoop; options: {}\n2022-11-22T01:34:52.712Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode with extender: EnvSinglenode; options: {}\n2022-11-22T01:34:52.712Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode with extender: ConfigDefault; options: {}\n2022-11-22T01:34:52.714Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-no-impersonation with extender: Standard; options: {}\n2022-11-22T01:34:52.715Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-no-impersonation with extender: Hadoop; options: {}\n2022-11-22T01:34:52.724Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-no-impersonation with extender: HadoopKerberos; options: {}\n2022-11-22T01:34:52.730Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-no-impersonation with extender: EnvSinglenodeKerberosHdfsNoImpersonation; options: {}\n2022-11-22T01:34:52.731Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-no-impersonation with extender: ConfigDefault; options: {}\n2022-11-22T01:34:52.733Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-no-impersonation-with-credential-cache with extender: Standard; options: {}\n2022-11-22T01:34:52.733Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-no-impersonation-with-credential-cache with extender: Hadoop; options: {}\n2022-11-22T01:34:52.734Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-no-impersonation-with-credential-cache with extender: HadoopKerberos; options: {}\n2022-11-22T01:34:52.735Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-no-impersonation-with-credential-cache with extender: EnvSinglenodeKerberosHiveNoImpersonationWithCredentialCache; options: {}\n2022-11-22T01:34:52.735Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-no-impersonation-with-credential-cache with extender: ConfigDefault; options: {}\n2022-11-22T01:34:52.736Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-hdfs-impersonation with extender: Standard; options: {}\n2022-11-22T01:34:52.737Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-hdfs-impersonation with extender: Hadoop; options: {}\n2022-11-22T01:34:52.738Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-hdfs-impersonation with extender: EnvSinglenodeHdfsImpersonation; options: {}\n2022-11-22T01:34:52.738Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-hdfs-impersonation with extender: ConfigDefault; options: {}\n2022-11-22T01:34:52.739Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation with extender: Standard; options: {}\n2022-11-22T01:34:52.740Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation with extender: Hadoop; options: {}\n2022-11-22T01:34:52.741Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation with extender: HadoopKerberos; options: {}\n2022-11-22T01:34:52.741Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation with extender: EnvSinglenodeKerberosHdfsImpersonation; options: {}\n2022-11-22T01:34:52.741Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation with extender: ConfigDefault; options: {}\n2022-11-22T01:34:52.743Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls with extender: Standard; options: {}\n2022-11-22T01:34:52.743Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls with extender: Hadoop; options: {}\n2022-11-22T01:34:52.744Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls with extender: EnvMultinodeTls; options: {}\n2022-11-22T01:34:52.748Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls with extender: ConfigDefault; options: {}\n2022-11-22T01:34:52.749Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos with extender: Standard; options: {}\n2022-11-22T01:34:52.750Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos with extender: Hadoop; options: {}\n2022-11-22T01:34:52.751Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos with extender: HadoopKerberos; options: {}\n2022-11-22T01:34:52.751Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos with extender: EnvMultinodeTlsKerberos; options: {}\n2022-11-22T01:34:53.351Z\tINFO\tmain\torg.testcontainers.dockerclient.DockerClientProviderStrategy\tFound Docker environment with local Unix socket (unix:///var/run/docker.sock)\n2022-11-22T01:34:53.364Z\tINFO\tmain\torg.testcontainers.DockerClientFactory\tDocker host IP address is localhost\n2022-11-22T01:34:53.385Z\tINFO\tmain\torg.testcontainers.DockerClientFactory\tConnected to docker: \n  Server Version: 20.10.20+azure-1\n  API Version: 1.41\n  Operating System: Ubuntu 20.04.5 LTS\n  Total Memory: 6944 MB\n2022-11-22T01:34:53.434Z\tINFO\tmain\t🐳 [testcontainers/ryuk:0.3.3]\tPulling docker image: testcontainers/ryuk:0.3.3. Please be patient; this may take some time but only needs to be done once.\n2022-11-22T01:34:53.745Z\tINFO\tdocker-java-stream-288019201\t🐳 [testcontainers/ryuk:0.3.3]\tStarting to pull image\n2022-11-22T01:34:53.762Z\tINFO\tdocker-java-stream-288019201\t🐳 [testcontainers/ryuk:0.3.3]\tPulling image layers:  0 pending,  0 downloaded,  0 extracted, (0 bytes/0 bytes)\n2022-11-22T01:34:53.860Z\tINFO\tdocker-java-stream-288019201\t🐳 [testcontainers/ryuk:0.3.3]\tPulling image layers:  2 pending,  1 downloaded,  0 extracted, (57 KB/? MB)\n2022-11-22T01:34:53.877Z\tINFO\tdocker-java-stream-288019201\t🐳 [testcontainers/ryuk:0.3.3]\tPulling image layers:  1 pending,  2 downloaded,  0 extracted, (57 KB/? MB)\n2022-11-22T01:34:53.900Z\tINFO\tdocker-java-stream-288019201\t🐳 [testcontainers/ryuk:0.3.3]\tPulling image layers:  0 pending,  3 downloaded,  0 extracted, (60 KB/5 MB)\n2022-11-22T01:34:54.094Z\tINFO\tdocker-java-stream-288019201\t🐳 [testcontainers/ryuk:0.3.3]\tPulling image layers:  0 pending,  3 downloaded,  1 extracted, (2 MB/5 MB)\n2022-11-22T01:34:54.225Z\tINFO\tdocker-java-stream-288019201\t🐳 [testcontainers/ryuk:0.3.3]\tPulling image layers:  0 pending,  3 downloaded,  2 extracted, (2 MB/5 MB)\n2022-11-22T01:34:54.336Z\tINFO\tdocker-java-stream-288019201\t🐳 [testcontainers/ryuk:0.3.3]\tPulling image layers:  0 pending,  3 downloaded,  3 extracted, (5 MB/5 MB)\n2022-11-22T01:34:54.366Z\tINFO\tmain\t🐳 [testcontainers/ryuk:0.3.3]\tCreating container for image: testcontainers/ryuk:0.3.3\n2022-11-22T01:34:54.447Z\tINFO\tmain\t🐳 [testcontainers/ryuk:0.3.3]\tContainer testcontainers/ryuk:0.3.3 is starting: 22d71adb6a6e3f7bb55dd20239e2aaf275ad117a67642274bf327d680d55ea88\n2022-11-22T01:34:54.939Z\tINFO\tmain\t🐳 [testcontainers/ryuk:0.3.3]\tContainer testcontainers/ryuk:0.3.3 started in PT1.545316668S\n2022-11-22T01:34:54.945Z\tINFO\tmain\torg.testcontainers.utility.RyukResourceReaper\tRyuk started - will monitor and terminate Testcontainers containers on JVM exit\n2022-11-22T01:34:54.945Z\tINFO\tmain\torg.testcontainers.DockerClientFactory\tChecking the system...\n2022-11-22T01:34:54.945Z\tINFO\tmain\torg.testcontainers.DockerClientFactory\t✔︎ Docker server version should be at least 1.6.0\n2022-11-22T01:34:54.947Z\tINFO\tmain\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling docker image: ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72. Please be patient; this may take some time but only needs to be done once.\n2022-11-22T01:34:55.206Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tStarting to pull image\n2022-11-22T01:34:55.207Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending,  0 downloaded,  0 extracted, (0 bytes/0 bytes)\n2022-11-22T01:34:55.325Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 35 pending,  1 downloaded,  0 extracted, (1 KB/? MB)\n2022-11-22T01:34:55.453Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 34 pending,  2 downloaded,  0 extracted, (19 MB/? MB)\n2022-11-22T01:34:55.602Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 33 pending,  3 downloaded,  0 extracted, (51 MB/? MB)\n2022-11-22T01:34:55.810Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 32 pending,  4 downloaded,  0 extracted, (118 MB/? MB)\n2022-11-22T01:34:55.956Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 31 pending,  5 downloaded,  0 extracted, (149 MB/? MB)\n2022-11-22T01:34:56.223Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 30 pending,  6 downloaded,  0 extracted, (217 MB/? MB)\n2022-11-22T01:34:56.445Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 29 pending,  7 downloaded,  0 extracted, (275 MB/? MB)\n2022-11-22T01:34:56.825Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 28 pending,  8 downloaded,  0 extracted, (392 MB/? MB)\n2022-11-22T01:34:57.015Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 27 pending,  9 downloaded,  0 extracted, (447 MB/? MB)\n2022-11-22T01:34:57.212Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 26 pending, 10 downloaded,  0 extracted, (505 MB/? MB)\n2022-11-22T01:34:57.342Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 25 pending, 11 downloaded,  0 extracted, (530 MB/? MB)\n2022-11-22T01:34:57.425Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 24 pending, 12 downloaded,  0 extracted, (551 MB/? MB)\n2022-11-22T01:34:57.522Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 23 pending, 13 downloaded,  0 extracted, (573 MB/? MB)\n2022-11-22T01:34:57.615Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 22 pending, 14 downloaded,  0 extracted, (592 MB/? MB)\n2022-11-22T01:34:57.967Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 21 pending, 15 downloaded,  0 extracted, (667 MB/? MB)\n2022-11-22T01:34:58.176Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 20 pending, 16 downloaded,  0 extracted, (721 MB/? MB)\n2022-11-22T01:34:58.335Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 19 pending, 17 downloaded,  0 extracted, (773 MB/? MB)\n2022-11-22T01:34:58.605Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 18 pending, 18 downloaded,  0 extracted, (833 MB/? MB)\n2022-11-22T01:34:58.780Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 17 pending, 19 downloaded,  0 extracted, (874 MB/? MB)\n2022-11-22T01:34:58.931Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 16 pending, 20 downloaded,  0 extracted, (930 MB/? MB)\n2022-11-22T01:34:59.107Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 15 pending, 21 downloaded,  0 extracted, (955 MB/? MB)\n2022-11-22T01:34:59.200Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 14 pending, 22 downloaded,  0 extracted, (981 MB/? MB)\n2022-11-22T01:34:59.262Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 13 pending, 23 downloaded,  0 extracted, (994 MB/? MB)\n2022-11-22T01:34:59.365Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 12 pending, 24 downloaded,  0 extracted, (1007 MB/? MB)\n2022-11-22T01:34:59.452Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 11 pending, 25 downloaded,  0 extracted, (1015 MB/? MB)\n2022-11-22T01:34:59.538Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers: 10 pending, 26 downloaded,  0 extracted, (1015 MB/? MB)\n2022-11-22T01:34:59.601Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  9 pending, 27 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:34:59.674Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  8 pending, 28 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:34:59.712Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  7 pending, 29 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:34:59.802Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  6 pending, 30 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:34:59.829Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  5 pending, 31 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:34:59.921Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  4 pending, 32 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:34:59.925Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  3 pending, 33 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:35:00.041Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  2 pending, 34 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:35:00.052Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  1 pending, 35 downloaded,  0 extracted, (1 GB/? MB)\n2022-11-22T01:35:05.042Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  0 extracted, (1 GB/1 GB)\n2022-11-22T01:35:06.766Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  1 extracted, (1 GB/1 GB)\n2022-11-22T01:35:07.232Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  2 extracted, (1 GB/1 GB)\n2022-11-22T01:35:15.828Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  3 extracted, (1 GB/1 GB)\n2022-11-22T01:35:15.898Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  4 extracted, (1 GB/1 GB)\n2022-11-22T01:35:15.959Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  5 extracted, (1 GB/1 GB)\n2022-11-22T01:35:32.936Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  6 extracted, (1 GB/1 GB)\n2022-11-22T01:35:32.999Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  7 extracted, (1 GB/1 GB)\n2022-11-22T01:35:33.082Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  8 extracted, (1 GB/1 GB)\n2022-11-22T01:35:33.385Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded,  9 extracted, (1 GB/1 GB)\n2022-11-22T01:35:33.724Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 10 extracted, (1 GB/1 GB)\n2022-11-22T01:35:33.780Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 11 extracted, (1 GB/1 GB)\n2022-11-22T01:35:33.845Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 12 extracted, (1 GB/1 GB)\n2022-11-22T01:35:33.912Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 13 extracted, (1 GB/1 GB)\n2022-11-22T01:35:34.013Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 14 extracted, (1 GB/1 GB)\n2022-11-22T01:35:34.076Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 15 extracted, (1 GB/1 GB)\n2022-11-22T01:35:38.335Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 16 extracted, (1 GB/1 GB)\n2022-11-22T01:35:38.694Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 17 extracted, (1 GB/1 GB)\n2022-11-22T01:35:38.760Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 18 extracted, (1 GB/1 GB)\n2022-11-22T01:35:38.817Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 19 extracted, (1 GB/1 GB)\n2022-11-22T01:35:38.886Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 20 extracted, (1 GB/1 GB)\n2022-11-22T01:35:38.946Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 21 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.025Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 22 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.079Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 23 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.139Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 24 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.196Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 25 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.249Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 26 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.308Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 27 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.369Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 28 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.425Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 29 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.482Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 30 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.541Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 31 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.599Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 32 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.656Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 33 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.712Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 34 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.766Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 35 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.824Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPulling image layers:  0 pending, 36 downloaded, 36 extracted, (1 GB/1 GB)\n2022-11-22T01:35:39.834Z\tINFO\tdocker-java-stream-1890065369\t🐳 [ghcr.io/trinodb/testing/hdp2.6-hive-kerberized:72]\tPull complete. 36 layers, pulled in 44s (downloaded 1 GB at 40 MB/s)\n2022-11-22T01:35:39.840Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos with extender: ConfigDefault; options: {}\n2022-11-22T01:35:39.842Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-wire-encryption with extender: Standard; options: {}\n2022-11-22T01:35:39.842Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-wire-encryption with extender: Hadoop; options: {}\n2022-11-22T01:35:39.843Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-wire-encryption with extender: HadoopKerberos; options: {}\n2022-11-22T01:35:39.843Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-wire-encryption with extender: EnvSinglenodeKerberosHdfsImpersonationWithWireEncryption; options: {}\n2022-11-22T01:35:39.844Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-wire-encryption with extender: ConfigDefault; options: {}\n2022-11-22T01:35:39.846Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-data-protection with extender: Standard; options: {}\n2022-11-22T01:35:39.846Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-data-protection with extender: Hadoop; options: {}\n2022-11-22T01:35:39.847Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-data-protection with extender: HadoopKerberos; options: {}\n2022-11-22T01:35:39.847Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-data-protection with extender: EnvSinglenodeKerberosHdfsImpersonationWithDataProtection; options: {}\n2022-11-22T01:35:39.848Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hdfs-impersonation-with-data-protection with extender: ConfigDefault; options: {}\n2022-11-22T01:35:39.849Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos-delegation with extender: Standard; options: {}\n2022-11-22T01:35:39.849Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos-delegation with extender: Hadoop; options: {}\n2022-11-22T01:35:39.850Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos-delegation with extender: HadoopKerberos; options: {}\n2022-11-22T01:35:39.850Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos-delegation with extender: EnvMultinodeTlsKerberosDelegation; options: {}\n2022-11-22T01:35:39.852Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-tls-kerberos-delegation with extender: ConfigDefault; options: {}\n2022-11-22T01:35:39.853Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-hive-impersonation with extender: Standard; options: {}\n2022-11-22T01:35:39.854Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-hive-impersonation with extender: Hadoop; options: {}\n2022-11-22T01:35:39.854Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-hive-impersonation with extender: EnvSinglenodeHiveImpersonation; options: {}\n2022-11-22T01:35:39.854Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-hive-impersonation with extender: ConfigDefault; options: {}\n2022-11-22T01:35:39.856Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation with extender: Standard; options: {}\n2022-11-22T01:35:39.856Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation with extender: Hadoop; options: {}\n2022-11-22T01:35:39.857Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation with extender: HadoopKerberos; options: {}\n2022-11-22T01:35:39.857Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation with extender: EnvSinglenodeKerberosHiveImpersonation; options: {}\n2022-11-22T01:35:39.857Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation with extender: ConfigDefault; options: {}\n2022-11-22T01:35:39.858Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation-with-credential-cache with extender: Standard; options: {}\n2022-11-22T01:35:39.859Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation-with-credential-cache with extender: Hadoop; options: {}\n2022-11-22T01:35:39.859Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation-with-credential-cache with extender: HadoopKerberos; options: {}\n2022-11-22T01:35:39.860Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation-with-credential-cache with extender: EnvSinglenodeKerberosHiveImpersonationWithCredentialCache; options: {}\n2022-11-22T01:35:39.860Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-kerberos-hive-impersonation-with-credential-cache with extender: ConfigDefault; options: {}\n2022-11-22T01:35:39.861Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-hive-caching with extender: Standard; options: {}\n2022-11-22T01:35:39.861Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-hive-caching with extender: Hadoop; options: {}\n2022-11-22T01:35:39.862Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-hive-caching with extender: EnvMultinodeHiveCaching; options: {}\n2022-11-22T01:35:39.863Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment multinode-hive-caching with extender: ConfigDefault; options: {}\n2022-11-22T01:35:39.864Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-delta-lake-databricks73 with extender: Standard; options: {}\n2022-11-22T01:35:39.865Z\tINFO\tmain\tio.trino.tests.product.launcher.env.EnvironmentProvider\tBuilding environment singlenode-delta-lake-databricks73 with extender: EnvSinglenodeDeltaLakeDatabricks73; options: {}\n2022-11-22T01:35:39.867Z\tINFO\tmain\tstderr\tjava.lang.RuntimeException: java.lang.IllegalStateException: Environment variable DATABRICKS_AWS_ACCESS_KEY_ID not set\n\tat io.trino.tests.product.launcher.cli.Commands.runCommand(Commands.java:44)\n\tat io.trino.tests.product.launcher.cli.SuiteDescribe.call(SuiteDescribe.java:86)\n\tat io.trino.tests.product.launcher.cli.SuiteDescribe.call(SuiteDescribe.java:58)\n\tat picocli.CommandLine.executeUserObject(CommandLine.java:2041)\n\tat picocli.CommandLine.access$1500(CommandLine.java:148)\n\tat picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)\n\tat picocli.CommandLine$RunLast.handle(CommandLine.java:2453)\n\tat picocli.CommandLine$RunLast.handle(CommandLine.java:2415)\n\tat picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)\n\tat picocli.CommandLine$RunLast.execute(CommandLine.java:2417)\n\tat picocli.CommandLine.execute(CommandLine.java:2170)\n\tat io.trino.tests.product.launcher.cli.Launcher.run(Launcher.java:73)\n\tat io.trino.tests.product.launcher.cli.Launcher.main(Launcher.java:62)\nCaused by: java.lang.IllegalStateException: Environment variable DATABRICKS_AWS_ACCESS_KEY_ID not set\n\tat io.trino.tests.product.launcher.env.environment.AbstractSinglenodeDeltaLakeDatabricks.exportAWSCredential(AbstractSinglenodeDeltaLakeDatabricks.java:89)\n\tat io.trino.tests.product.launcher.env.environment.AbstractSinglenodeDeltaLakeDatabricks.exportAWSCredentials(AbstractSinglenodeDeltaLakeDatabricks.java:79)\n\tat io.trino.tests.product.launcher.env.environment.AbstractSinglenodeDeltaLakeDatabricks.lambda$extendEnvironment$0(AbstractSinglenodeDeltaLakeDatabricks.java:56)\n\tat io.trino.tests.product.launcher.env.Environment$Builder.configureContainer(Environment.java:521)\n\tat io.trino.tests.product.launcher.env.environment.AbstractSinglenodeDeltaLakeDatabricks.extendEnvironment(AbstractSinglenodeDeltaLakeDatabricks.java:56)\n\tat io.trino.tests.product.launcher.env.common.EnvironmentExtender.extendEnvironment(EnvironmentExtender.java:35)\n\tat io.trino.tests.product.launcher.env.EnvironmentProvider.extend(EnvironmentProvider.java:83)\n\tat io.trino.tests.product.launcher.env.EnvironmentProvider.lambda$createEnvironment$0(EnvironmentProvider.java:59)\n\tat com.google.common.collect.ImmutableList.forEach(ImmutableList.java:422)\n\tat io.trino.tests.product.launcher.env.EnvironmentProvider.createEnvironment(EnvironmentProvider.java:59)\n\tat io.trino.tests.product.launcher.env.EnvironmentFactory.get(EnvironmentFactory.java:42)\n\tat io.trino.tests.product.launcher.cli.SuiteDescribe$Execution.call(SuiteDescribe.java:255)\n\tat io.trino.tests.product.launcher.cli.SuiteDescribe$Execution.call(SuiteDescribe.java:207)\n\tat io.trino.tests.product.launcher.cli.Commands.runCommand(Commands.java:41)\n\t... 12 more\n\n2022-11-22T01:35:39.868Z\tINFO\tThread-1\tio.airlift.bootstrap.LifeCycleManager\tJVM is shutting down, cleaning up\n2022-11-22T01:35:39.868Z\tINFO\tThread-1\tio.airlift.bootstrap.LifeCycleManager\tLife cycle stopping...\n")
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -776,8 +776,8 @@ jobs:
           ABFS_ACCESS_KEY: ${{ secrets.AZURE_ABFS_ACCESSKEY }}
           S3_BUCKET: trino-ci-test
           AWS_REGION: us-east-2
-          DATABRICKS_AWS_ACCESS_KEY_ID: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
-          DATABRICKS_AWS_SECRET_ACCESS_KEY: ${{ secrets.TRINO_AWS_SECRET_ACCESS_KEY }}
+          TRINO_AWS_ACCESS_KEY_ID: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
+          TRINO_AWS_SECRET_ACCESS_KEY: ${{ secrets.TRINO_AWS_SECRET_ACCESS_KEY }}
           DATABRICKS_73_JDBC_URL: ${{ secrets.DATABRICKS_73_JDBC_URL }}
           DATABRICKS_91_JDBC_URL: ${{ secrets.DATABRICKS_91_JDBC_URL }}
           DATABRICKS_104_JDBC_URL: ${{ secrets.DATABRICKS_104_JDBC_URL }}

--- a/plugin/trino-delta-lake/README.md
+++ b/plugin/trino-delta-lake/README.md
@@ -169,8 +169,8 @@ environment for Trino:
 ```
 env S3_BUCKET=my-s3-bucket \
     AWS_REGION=us-east-2 \
-    DATABRICKS_AWS_SECRET_ACCESS_KEY=xxx \
-    DATABRICKS_AWS_ACCESS_KEY_ID=xxx \
+    TRINO_AWS_SECRET_ACCESS_KEY=xxx \
+    TRINO_AWS_ACCESS_KEY_ID=xxx \
     DATABRICKS_91_JDBC_URL='xxx' \
     DATABRICKS_LOGIN=token \
     DATABRICKS_TOKEN=xxx \

--- a/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/AbstractSinglenodeDeltaLakeDatabricks.java
+++ b/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/AbstractSinglenodeDeltaLakeDatabricks.java
@@ -76,9 +76,9 @@ public abstract class AbstractSinglenodeDeltaLakeDatabricks
 
     private DockerContainer exportAWSCredentials(DockerContainer container)
     {
-        container = exportAWSCredential(container, "DATABRICKS_AWS_ACCESS_KEY_ID", "AWS_ACCESS_KEY_ID", true);
-        container = exportAWSCredential(container, "DATABRICKS_AWS_SECRET_ACCESS_KEY", "AWS_SECRET_ACCESS_KEY", true);
-        return exportAWSCredential(container, "DATABRICKS_AWS_SESSION_TOKEN", "AWS_SESSION_TOKEN", false);
+        container = exportAWSCredential(container, "TRINO_AWS_ACCESS_KEY_ID", "AWS_ACCESS_KEY_ID", true);
+        container = exportAWSCredential(container, "TRINO_AWS_SECRET_ACCESS_KEY", "AWS_SECRET_ACCESS_KEY", true);
+        return exportAWSCredential(container, "TRINO_AWS_SESSION_TOKEN", "AWS_SESSION_TOKEN", false);
     }
 
     private DockerContainer exportAWSCredential(DockerContainer container, String credentialEnvVariable, String containerEnvVariable, boolean required)

